UKHO.PeriodicOutputService/UKHO.BESS.API.FunctionalTests/FunctionalTests/ScsIntegration/ScsIntegrationTests.cs

```diff
@@ -20,23 +20,15 @@ public async Task OneTimeSetupAsync()
         [Test]
         public async Task WhenICallScsEssDataEndpointWithValidToken_ThenSuccessStatusCode200IsReturned()
         {
-            HttpResponseMessage apiResponse = await ScsEndpointHelper.ScsEssDataEndpoint(testConfiguration.scsConfig.BaseUrl, ScsJwtToken);
+            HttpResponseMessage apiResponse = await ScsEndpointHelper.ScsEssDataEndpoint(testConfiguration.scsConfig.MockBaseUrl, ScsJwtToken);
```

Comment:
Method can be changed to "EssDataEndPoint"

Reply:
Resolved